db/seeds.rb

```diff
@@ -67,7 +68,8 @@
     next_question: 'Next question 6',
     is_root: false,
     flowchart_id: flowchart.id,
-    flowchart_node_id: 4
+    flowchart_node_id: 4,
+    is_leaf: true
```

Comment:
Can you merge master since `seeds.rb` was updated to include a few more nodes, so `is_leaf` will need to be added to different nodes.